pysoa/server/standalone.py

```diff
@@ -164,6 +165,20 @@ def django_main(server_getter):
     if args.settings:
         os.environ['DJANGO_SETTINGS_MODULE'] = args.settings
 
+    try:
+        # We have to import it manually, because we need to manipulate the settings before setup() is called, but we
+        # can't import django.conf.settings until after setup() is called.
+        django_settings = importlib.import_module(os.environ['DJANGO_SETTINGS_MODULE'])
+        django_settings.LOGGING = django_settings.SOA_SERVER_SETTINGS['logging']
```

Comment:
so this is preserved even if module is imported again by other code?

Reply:
Yes.